explore-analyze/visualize/charts/metric-charts.md

```diff
@@ -0,0 +1,292 @@
+---
+navigation_title: Metric charts
+applies_to:
+  stack: ga
+  serverless: ga
+description: Instructions and best practices for building metric charts with Kibana Lens in Elastic.
+---
+
+# Build metric charts with {{kib}}
+
+Metric charts make important single values stand out on a dashboard. They're perfect for highlighting KPIs such as error rates or SLOs for example, and for making them understandable at a glance with dynamic coloring or trend indicators.
+
+They work with any numeric data: plain numbers, percentages, or calculations like a count, sum, or average. You can get this numeric data from numeric fields stored in your {{es}} documents, or from aggregation functions and formulas that you can apply to any type of field.
+
+The best way to create metric charts in {{kib}} is with **Lens**.
```

Comment:
Let's say first and only! We don't want to promote any other metric charts like the one available in TSVB, so this is the only way to create metric charts in kibana.